### ISSUE TITLE
Return results without parsing it

### ISSUE THREAD

**Issue by ColinFay** [neo4j-rstats/neo4r#62]
There should be a option to not wrangle the results and return it as a list.

**Comment by usuallycwdillon** [comment]
+1 for this, @ColinFay 
More specifically, there should be an option to return results that can be piped directly into the next operation, e.g., mutate().
```
resultTibble <- <query> %>% 
  call_neo4j(con, type='row') %>% 
  mutate(
    fit = map(., ~ lm(x ~ y), data = .x)),
    tidied = map(fit, tidy),
    glanced = map(fit, glance),
    augmented = map(fit, augment) 
)
```

**Comment by ColinFay** [comment]
Closed via #86
